docs/journalist/starting_client.rst

```diff
@@ -1,16 +1,74 @@
-Starting the SecureDrop client
+Starting the SecureDrop Client
 ==============================
 
 .. include:: ../includes/top-warning.rst
 
+Double-click on the **SecureDrop Workstation Launcher** desktop shortcut to
+launch the SecureDrop Client application.
+
+.. | screenshot_desktop-shortcut|
+
 Performing updates
 ------------------
 
+When you launch the application, it will first check for security
+updates. This can take several minutes.
+
+.. | screenshot_check_updates |
+
+If updates are found, you will be asked to apply them before launching the
+client. You have the option of completing this now, or at another time. Keep in
```

Comment:
Consistency: "Client" is upper case elsewhere

Reply:
changed in 5eda33f